packages/@o3r/schematics/src/utility/package-manager-runner.ts

```diff
@@ -1,39 +1,64 @@
 import * as commentJson from 'comment-json';
+import { WorkspaceSchema } from '../interfaces';
 
 /** Npm packageManager */
 export type SupportedPackageManagers = 'npm' | 'yarn';
 
+/**
+ * Determine if the given packager manager is supported
+ *
+ * @param name Name of the package manager
+ */
+export function isSupportedPackageManager(name?: any): name is SupportedPackageManagers {
+  return name === 'yarn' || name === 'npm';
+}
+
 /**
  * Get the Package Manager
  *
  * @param enforcedNpmManager package manager to enforce
  */
-export function getPackageManagerName(enforcedNpmManager?: SupportedPackageManagers | '' | undefined | null): SupportedPackageManagers {
-  return enforcedNpmManager || (process.env?.npm_execpath?.includes('yarn') ? 'yarn' : 'npm');
+function getPackageManagerName(enforcedNpmManager?: SupportedPackageManagers): SupportedPackageManagers {
```

Comment:
`export` the function to stay not breaking

Reply:
Not breaking if committed before 9.0.0